src/browser/web-viewer.js

```diff
@@ -100,10 +155,15 @@ define((require, exports, module) => {
   WebViewer.onTitleChange = webViewerCursor => event =>
     webViewerCursor.set('title', event.detail);
 
+
   WebViewer.onLocationChange = webViewerCursor => event => {
-    webViewerCursor.merge(Object.assign({location: event.detail,
-                                        userInput: event.detail},
-                                        getHardcodedColors(event.detail)));
+    webViewerCursor.merge({location: event.detail,
+                           userInput: event.detail}).
+                    merge(getHardcodedColors(event.detail));
+
+    requestThumbnail(event.target).
+      then(WebViewer.onThumbnailChanged(webViewerCursor),
+           e => console.error(e));
```

Comment:
> I think this is the abort promise that fails (as expected).

Yes while debugging I was looking at when promise would fail and why, I'll remove that.